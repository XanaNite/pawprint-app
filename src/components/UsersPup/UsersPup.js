import React from 'react'
import './UsersPup.css'

export default function UsersPup(props){
    return (
        <>
            <li>
                <img src={props.img} alt='placeholder pup' />
                <p>{props.pup_name}</p>
            </li>
        </>
    )
}