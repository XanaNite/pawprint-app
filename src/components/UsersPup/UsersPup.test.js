import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import UsersPup from './UsersPup';

describe(`UsersPup component`, () => {
  const props = {
    className: 'test-class-name',
    children: <p>test children</p>,
    'data-other': 'test-other-prop'
  };

  it('renders UsersPup by default', () => {
    const wrapper = shallow(<UsersPup />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });

  it('renders the UsersPup given props', () => {
    const wrapper = shallow(<UsersPup {...props} />)
    expect(toJson(wrapper)).toMatchSnapshot()
  });
})
