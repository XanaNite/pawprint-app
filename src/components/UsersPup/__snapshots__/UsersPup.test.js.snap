// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersPup component renders UsersPup by default 1`] = `
<Fragment>
  <li
    className="pups"
  >
    <Link
      className="yourPup-link"
      to="/profile/undefined"
    >
      <img
        alt="placeholder pup"
      />
      <p />
    </Link>
  </li>
</Fragment>
`;

exports[`UsersPup component renders the UsersPup given props 1`] = `
<Fragment>
  <li
    className="pups"
  >
    <Link
      className="yourPup-link"
      to="/profile/undefined"
    >
      <img
        alt="placeholder pup"
      />
      <p />
    </Link>
  </li>
</Fragment>
`;
